api/filter.py

```diff
@@ -0,0 +1,34 @@
+from django_filters.rest_framework import FilterSet, filters
+from api.models import *
+
+
+class WeekdayFilter(FilterSet):
+    weekday = filters.CharFilter(method='course_day_filter')
+
+    class Meta:
+        model = Course
+        fields = ['weekday']
+
+    def course_day_filter(self):
```

Comment:
구글링해서 용례를 살펴보시면, 
```suggestion
    def course_day_filter(self, queryset, name, value):
```
와 같이 되어있을 거에요. 이렇게 하면 에러가 납니다😢